source/nodejs/adaptivecards/src/card-object.ts

```diff
@@ -136,6 +145,19 @@ export abstract class CardObject extends SerializableObject {
         return result;
     }
 
+    /**
+     * Finds the innermost CardObject who's renderedElement is equal to or parent of the query Node.
+     *
+     * @param node query Node
+     *
+     * @returns CardObject containing node, or undefined if not in tree.
+     */
+    findObject(node: Node): CardObject | undefined {
+        // default implementation for non-heirarchical CardObjects
```

Comment:
heirarchical -> hierarchical

Not sure this comment needs to exist, as I'm not sure a reader would understand what a "non-hierarchical CardObject" is